bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/link/ItemChannelLinkRegistry.java

```diff
@@ -90,31 +87,6 @@ public ItemChannelLink update(ItemChannelLink element) {
         return linkedItems;
     }
 
-    /**
-     * Returns a set of bound things for the given item name.
-     *
-     * @deprecated Will be removed soon. Use {@link ItemThingLinkRegistry#getLinkedThings(String)} instead.
-     *
-     * @param itemName
-     *            item name
-     * @return set of bound things for the given item name
-     */
-    @Deprecated
```

Comment:
> We still support item thing links?

No, we don't. This method is actually a shortcut for getting all linked channels and getting the set of things that these channels belong to. I personally don't have any use case for such a method, but I also wouldn't mind too much to keep it in place, if someone really needs it.